### PR TITLE
Update deployment config and script so they can be used for custom DC

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: deploy/overlays/custom_kustomization.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -custom
+namespace: website
+
+resources:
+  - ../base
+
+configMapGenerator:
```
Comment (Fructokinase) [inline deploy/overlays/custom_kustomization.yaml.tpl:25]: Shall we disable config map suffix from now on? 

```
generatorOptions:
 disableNameSuffixHash: true
```

File: deploy/overlays/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC
```
Comment (Fructokinase) [inline deploy/overlays/kustomization.yaml:1]: Was this meant to be under deploy/overlays/buoyant-country-377016/kustomization.yaml?

File: scripts/push_image.sh [hunk 3]
```diff
@@ -17,12 +17,14 @@
 
 set -e
 
+PROJECT_ID=$1
```
Comment (Fructokinase) [inline scripts/push_image.sh:20]: nit: add a check here?

File: deploy/overlays/custom_kustomization.yaml.tpl (hunk 1 above)
Comment (shifucun) [reply deploy/overlays/custom_kustomization.yaml.tpl:28]: Done

File: deploy/overlays/kustomization.yaml (hunk 2 above)
Comment (shifucun) [reply deploy/overlays/kustomization.yaml]: This should not be checked in. Removed it.

File: scripts/push_image.sh (hunk 3 above)
Comment (shifucun) [reply scripts/push_image.sh:20]: Done

**Pull request merged**